tests/jobs/test_triggerer_job.py

```diff
@@ -51,6 +52,24 @@
 pytestmark = pytest.mark.db_test
 
 
+@pytest.fixture(autouse=True)
```

Comment:
```suggestion
@pytest.fixture(scope="module", autouse=True)
```

It doesn’t feel right to do this for _all_ tests. Either this or we want to manually apply this to tests that matter instead.